tests/includes/Sync/Test_Collection_Rules_Plan.php

```
@@ -0,0 +1,290 @@
+	}
+
+	/**
+	 * A signature of the global filter table, used to detect leaked callbacks.
+	 *
+	 * @return array<string, int>
+	 */
+	private function hook_signature(): array {
+		$signature = array();
+		foreach ( $GLOBALS['wp_filter'] as $hook => $callbacks ) {
+			$count = 0;
+			foreach ( $callbacks->callbacks as $priority => $entries ) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$priority'. (undefined)

(UnusedLocalVariable)